invenio_stats/aggregations.py

```diff
@@ -293,7 +295,12 @@ def agg_iter(self, lower_limit, upper_limit):
         terms = histogram.bucket(
             'terms', 'terms', field=self.field,
             size=get_bucket_size(
-                self.client, self.event_index, self.field)
+                self.client,
+                self.event_index,
+                self.field,
+                start_date,
```

Comment:
```suggestion
                start_date=start_date,
```
For future safety, in case the signature of `get_bucket_size` changes